nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/process-group-configuration.css

```diff
@@ -87,7 +95,6 @@
 }
 
 #parameter-contexts-recursive-container {
-    float: left;
```

Comment:
Removal of the float here is good. With its removal though, we no longer need to float `process-group-parameter-context-combo` (the element before it) so we can remove that style too and we can remove the `div.clear` that comes after it.

Reply:
Thanks @mcgilman. I implemented your recommendation, but the next line "Execution Engine" appeared in-line with "Apply recursively" when I removed `<div class="clear"></div>`.  I believe you were referring to removing this at Line 50, right? So, I left this in place, but I removed the float for `process-group-parameter-context-combo`

Reply:
I'll take a look. But if nothing is floating (which should be the case now) then there should be no need to clear. But maybe I missed something else.

Reply:
I see. It is because the elements inside `parameter-contexts-recursive-container` are floated left. It would probably be clearer if the `div.clear` were the last element in that element but it's ok. I'll merge this one. Thanks again for the PR!

Reply:
Thanks @mcgilman !

Would you mind taking a look/merging the PR for nifi-1.x as well so this can get into the 1.24.0 release?
#7844